src/test/polymer/polymer-core-feature_test.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from 'chai';
+
+import {Visitor} from '../../javascript/estree-visitor';
+import {JavaScriptParser} from '../../javascript/javascript-parser';
+import {PolymerCoreFeatureScanner} from '../../polymer/polymer-core-feature-scanner';
+
+suite('PolymerCoreFeatureScanner', () => {
+
+  test('scans _addFeature call expressions', async () => {
+    const js = `
+      /** Feature A */
+      Polymer.Base._addFeature({
+        methodA: function() {}
+      });
+
+      /** Feature B */
+      Polymer.Base._addFeature({
+        methodB: function() {}
+      });
+
+      /** Invalid feature */
+      Polymer.Base._addFeature(null);
+
+      /** Not a feature at all */
+      Polymer.Base._somethingElse({
+        methodX: function() {}
+      });
+    `;
+
+    const parser = new JavaScriptParser();
+    const scanner = new PolymerCoreFeatureScanner();
+    const doc = parser.parse(js, 'features.js');
+    const visit = (visitor: Visitor) => Promise.resolve(doc.visit([visitor]));
+    const {features} = await scanner.scan(doc, visit);
+
+    assert.lengthOf(features, 3);
+    const [a, b, c] = features;
+
+    // TODO(aomarks) Why aren't the comments stripped correctly?
+    assert.equal(a.description, '* Feature A ');
+    assert.lengthOf(a.warnings, 0);
+    assert.lengthOf(a.properties, 1);
+    assert.equal(a.properties[0].name, 'methodA');
```

Comment:
It's currently coming out as a PolymerProperty with type:Function. Maybe related to #647.